### PR TITLE
[webview-flutter] Added step 09: Working with JavaScript Channels

### PR DESCRIPTION
**Pull request by JDDV**
Added the 'Evaluating JavaScript' code for the webview_flutter codelab (codelab is WIP).
This step contains adding a button in the `AppBar`'s menu to print the user-agent line in the console using JavaScript.

## Pre-launch Checklist

- [x] I read the [Flutter Style Guide] _recently_, and have followed its advice.
- [x] I signed the [CLA].
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-devrel channel on [Discord].

[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[CLA]: https://cla.developers.google.com/
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

File: webview_flutter/step_09/lib/main.dart [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+            IconButton(
+              icon: const Icon(Icons.arrow_back_ios),
+              onPressed: !webViewReady
+                  ? null
+                  : () async {
+                      if (await controller!.canGoBack()) {
+                        await controller.goBack();
+                      } else {
+                        // ignore: deprecated_member_use
+                        Scaffold.of(context).showSnackBar(
```
Comment (domesticmouse) [inline webview_flutter/step_09/lib/main.dart:87]: I'm curious why you are using a deprecated method? With codelabs we show the latest and greatest approach. This is probably a blocker.

Comment (domesticmouse) [reply webview_flutter/step_09/lib/main.dart:87]: https://flutter.dev/docs/release/breaking-changes/scaffold-messenger is probably appropriate here.

**Review by domesticmouse — CHANGES_REQUESTED** [review]

File: webview_flutter/step_09/lib/main.dart (hunk 1 above)
Comment (JDDV) [reply webview_flutter/step_09/lib/main.dart:87]: Looks like we missed this, I will commit the changes for this and the other PR's as well. It looks like this is also the case for step 07 which is already merged, there will come a PR to update this as well.

File: webview_flutter/step_09/lib/main.dart [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+  final Completer<WebViewController> _controller =
+      Completer<WebViewController>();
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      home: Scaffold(
+        appBar: AppBar(
+          title: const Text('Flutter WebView example'),
+          // This drop down menu demonstrates that Flutter widgets can be shown over the web view.
```
Comment (domesticmouse) [inline webview_flutter/step_09/lib/main.dart:22]: Please delete these comments. If they are important, they should be in the codelab text. Thanks!

**Review by domesticmouse — COMMENTED** [review]
Migrate comments to the codelab text. Thanks!

**Review by domesticmouse — APPROVED** [review]

**Pull request merged**